apdfl18/DotNet/ver.html

```diff
@@ -43,7 +43,7 @@
             land = ""
         }
         if ((lang != "") || (lang != "-") || (ver != "") || (ver != "-")) {
-            window.top.location.href = "https://docs.datalogics.com/apdfl18/" + lang + "/" + ver + "/index.html"
+            window.top.location.href = "https://docs.datalogics.com/apdfl18/" + lang + "/" + ver + "/" + lang + "/index.html"
```

Comment:
We took out that extra `+ lang` recently because a long time ago we moved removed the directory